android/build.gradle

```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.library'
 
 android {
     namespace "com.remobile.cordova"
-    compileSdkVersion 23
-    buildToolsVersion "30.0.3"
+    compileSdkVersion 34
```

Comment:
I think this should be kept in sync with https://github.com/IndoorAtlas/cordova-plugin/blob/react-native/src/android/build.gradle#L5 i.e. 31 currently.